### PR TITLE
[BEEEP] Update development and QA dashboard URLs for payment gateways

### PR DESCRIPTION
**Pull request by cturnbull-bitwarden**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [X] Other
```

## Objective

When attempting to open the payment gateway links for an organization or user in the Bitwarden Portal in QA or local, the links take you to the production environments for Stripe and Braintree. This PR addresses that by checking the hosting environment before generating the links

## Code changes

* **_OrganizationFormScripts.cshtml:** Updated the links to Stripe or Braintree to be the sandbox environment when in development or QA environments
* **Users\Edit.cshtml:** Updated the links to Stripe or Braintree to be the sandbox environment when in development or QA environments

## Before you submit

- Please check for formatting errors (`dotnet format --verify-no-changes`) (required)
- If making database changes - make sure you also update Entity Framework queries and/or migrations
- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

File: src/Admin/Views/Shared/_OrganizationFormScripts.cshtml [hunk 1]
```diff
@@ -15,10 +16,15 @@
                 return;
             }
             if (gateway.value === '@((byte)GatewayType.Stripe)') {
-                window.open('https://dashboard.stripe.com/customers/' + customerId.value, '_blank');
+                const url = '@(HostingEnvironment.IsDevelopment()
```
Comment (amorask-bitwarden) [inline src/Admin/Views/Shared/_OrganizationFormScripts.cshtml:19]: ⛏️ All looks good to me! Only thing I can see would be the potential to cut down on the repeated ternary operators with extension methods off the `IHostingEnvironment` like `IHostingEnvironment.GetStripeUrl` and `IHostingEnvironment.GetBraintreeMerchantUrl`.

**Review by amorask-bitwarden — APPROVED** [review]

File: src/Admin/Views/Shared/_OrganizationFormScripts.cshtml (hunk 1 above)
Comment (cturnbull-bitwarden) [reply src/Admin/Views/Shared/_OrganizationFormScripts.cshtml]: Thanks for the suggestion! I just pushed a change to add this in, so take a look and let me know what you think

File: src/Admin/Views/Users/Edit.cshtml [hunk 2]
```diff
@@ -45,10 +46,15 @@
                 }
 
                 if (gateway.value === '@((byte)Bit.Core.Enums.GatewayType.Stripe)') {
-                    window.open('https://dashboard.stripe.com/customers/' + customerId.value, '_blank');
+                    const url = '@(HostingEnvironment.IsDevelopment()
+                                     ? "https://dashboard.stripe.com/test"
```
Comment (cyprain-okeke) [inline src/Admin/Views/Users/Edit.cshtml:50]: We can move the urls to  variables and use the variables within the code so we have a single place to replace the pr for future changes.

Comment (cturnbull-bitwarden) [reply src/Admin/Views/Users/Edit.cshtml:50]: I pushed a change yesterday which should accomplish this! Let me know what you think

**Review by cyprain-okeke — APPROVED** [review]
looks good

**Review by amorask-bitwarden — APPROVED** [review]

**Review by cagonzalezcs — APPROVED** [review]

**Pull request merged**